### DISCUSSION TITLE
怎么设置中文

### DISCUSSION THREAD

**Discussion by weirdcore-phantom** [community/community#120856] [New to GitHub]
呃呃呃，刚接触到git hub，不知道怎么调整语言（

**Comment by gurjeetsinghvirdee** [comment]
hey @weirdcore-phantom 

![image](https://github.com/community/community/assets/73753957/0a89e069-7e4d-45c2-9f7b-81d47914f96c)

**Comment by ebndev** [comment]
Hi @weirdcore-phantom, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
